Containerfile

```diff
@@ -68,7 +68,7 @@ RUN microdnf -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install \
  && microdnf clean all
 
 # Remove gnutls and everything that depends on it. See https://github.com/skupperproject/skupper-router/issues/1477
-RUN microdnf -y remove gnutls glib2 gobject-introspection libpeas microdnf gnupg2 gpgme libdnf json-glib libmodulemd librepo librhsm
+RUN microdnf -y remove gnutls glib2 gobject-introspection libpeas microdnf gnupg2 gpgme libdnf json-glib libmodulemd librepo librhsm libsolv rpm rpm-libs libarchive 
```

Comment:
Should the comment be updated to say "libarchive and everything that depends on it"?

Reply:
I fixed the comment, thanks